app/views/users/profile.html.erb

```diff
@@ -233,12 +233,13 @@
       </div>
     </div>
   <% end %>
-
+ <%if !Rails.env.production?%>
```

Comment:
Just some quick comments about readability: 
- could you please add a space between the conditional and the tags?

Reply:
I have added the space, and thank you for the screenshot also.